src/Components/InputField/index.js

```diff
@@ -1,18 +1,41 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import * as yup from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+
+const schema = yup.object().shape({
+  taskTittle: yup
+    .string()
+    .required()
+    .max(35, "Task Tittle can contain 35 character only."),
+});
+
 export const InputField = ({ addTodo }) => {
```

Comment:
add more validations like task can't contain numbers and more you can come up with